Cargo.toml

```diff
@@ -11,7 +11,7 @@ authors = ["Apollo <opensource@apollographql.com>"]
 edition = "2024"
 license-file = "LICENSE"
 repository = "https://github.com/apollographql/apollo-mcp-server"
-rust-version = "1.89.0"
+rust-version = "1.90.0"
```

Comment:
Nit: So this is technically the minimally supported rust version (MSRV) [more details [here](https://doc.rust-lang.org/cargo/reference/rust-version.html)]. We had previously pinned this to `1.89.0` since we were using some of its newer features (let chaining in conditionals).

We can bump this as we bump our in-use version of rust, but let me know if you prefer otherwise.